processor/processor.go

```diff
@@ -1544,18 +1548,6 @@ type storeMessage struct {
 }
 
 func (proc *HandleT) Store(in storeMessage) {
-	// FIXME: This is a hack to get around the fact that,
```

Comment:
since we have introduced context in JobsDB, I believe this part is not required any more. 
https://github.com/rudderlabs/rudder-server/pull/1776